book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] This method is for creating a rental given a person and date. Since the rental is added upon creation, this method needs to take that into account. Kindly refactor the method to create a rental that associates this book with the person through rental.

  ```
  def add_rental(date, person)
     Rental.new(date, self, person)
  end
  ```